src/test/scala/com/fulcrumgenomics/sv/UnitSpec.scala

```diff
@@ -1,6 +1,24 @@
 package com.fulcrumgenomics.sv
 
-import org.scalatest.{FlatSpec, Matchers, OptionValues}
+import com.fulcrumgenomics.FgBioDef.PathToBam
+import com.fulcrumgenomics.bam.api.{SamRecord, SamSource}
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.{Matchers, OptionValues}
+
+import java.nio.file.{Files, Path}
 
 /** Base class for unit tests. */
-class UnitSpec extends FlatSpec with Matchers with OptionValues {}
+trait UnitSpec extends AnyFlatSpec with Matchers with OptionValues {
```

Comment:
we should really make `fgbio`'s `UnitSpec` public instead of copying everywhere

Reply:
If you can figure out a way to do that without having to add all the testing libraries to the non-test dependencies n each project I'm all for it.  Likely we just need to make an `fgscalatest` project and put stuff there so we can depened on it only in test configurations.